tests/pricing_models/test_pricing_model_utils.py

```diff
@@ -33,116 +32,119 @@ def run_calc_k_const_test(
                     init_share_price=1,  # u = 1
                 ),
                 "time_elapsed": 0.25,  # t = 0.25
-                # k = c/u * (u*y)**t + (2y+c*x)**t
-                #     = 1/1 * (1*500000)**0.25 + (2*500000+1*500000)**0.25
-                #     = 61.587834600530776
+                # k = c/u * (u*z)**t + (2y+c*z)**t
```

Comment:
update the `2y + cz` part to `y + s` no?  It's still approximated as 2y+cz but we should try to limit that in the comments as much as possible so we can move away from it.